acceptance/auth/bundle_and_profile/script.prepare

```diff
@@ -0,0 +1,9 @@
+DB_CFG="$HOME/.databrickscfg"
```

Comment:
$HOME is currently shared between all (parallel) test cases. So this will affect every other test.

For this case we should have a specially created tempdir for HOME. I suggest to add a new option to test config (IsolatedHome = true) that creates new tmpdir for home instead of reusing the common one.

Reply:
Actually we don't really need this as we can use DATABRICKS_CONFIG_FILE in script